Extensions/one_click_postage.css

```diff
@@ -11,6 +11,19 @@
     line-height: 10px;
 }
 
+.post.themed_posts_experiment .post_wrapper,
+.post.themed_posts_experiment.post_full .post_footer,
+.post.themed_posts_experiment .post_controls,
+.post.themed_posts_experiment .post_controls_inner,
+.post.themed_posts_experiment .reblogged {
+	background-color: inherit;
```

Comment:
@nightpool The original selector .post_controls .post_control.reblog.reblogged:after will set the tick to be a green colour. If I move the color: inherit; to it, it will inherit the black colour if the user does not have Tumblr Labs - Themed Posts enabled.